### ISSUE TITLE
Unable to converge 

### ISSUE THREAD

**Issue by simleopold** [schatzlab/genomescope#120]
Hi,

I'm a student working on a project to estimate the size of a species' genome. I generated a histo file from ONT Minion datas and genome scope was unable to converge. Does the problem come from the long reads ? 

Thanks for your help.

**Comment by mschatz** [comment]
Yes, ONT reads have a higher error rate that obscures the kmer profile. I
would suggest trying to run an assembly and then aligning the reads to the
assembled contigs so that you can estimate the average coverage. Then if
you sum up the lengths of all of your ONT reads and divide by the estimated
coverage you will get a very rough idea of the expected genome size. It
will be very rough but should get you an order of magnitude estimate, e.g.
if you have 100GB of reads, and your average coverage is 40x, you can
estimate the genome is 2.5GB.

Hope this helps

Mike

**Comment by simleopold** [comment]
Thank you for your help, do you know any tool capable of doing this ?

**Comment by mschatz** [comment]
I dont know of a push button solution. I would start with minimap2 to align
the reads to your contigs, and then bedtools or samtools to compute the
read coverage. From this I would plot a histogram of the reported coverage
(using R/python) to visually inspect what looks to be the average coverage.

Good luck!
Mike
